ActiveTheme.php

```diff
@@ -42,8 +43,11 @@ class ActiveTheme
      * @param array                           $themes
      * @param Helper\DeviceDetectionInterface $deviceDetection
      */
-    public function __construct($name, array $themes = array(), DeviceDetectionInterface $deviceDetection = null)
+    public function __construct($name, array $themes = array(), DeviceDetectionInterface $deviceDetection = null, ThemeLocator $themeLocator)
```

Comment:
Yep, you're right.
Since it's after DeviceDetectionInterface which is optional, it's better that way anyway.